bin/deploy.sh

```diff
@@ -4,6 +4,7 @@ set -e
 export INGRESS_INTERNAL_ANNOTATIONS=$HOF_CONFIG/ingress-internal-annotations.yaml
 export INGRESS_EXTERNAL_ANNOTATIONS=$HOF_CONFIG/ingress-external-annotations.yaml
 export CONFIGMAP_VALUES=$HOF_CONFIG/configmap-values.yaml
+export CONFIGMAP1_VALUES=$HOF_CONFIG/imscertchain-store.yaml
```

Comment:
Sorry, I meant that `CONFIGMAP1_VALUES` could be updated :)

Reply:
We have configmap_values already I guess this could be...?